lib/fluent/plugin/out_sumologic.rb

```diff
@@ -136,6 +143,10 @@ def configure(conf)
       conf['custom_fields'] = nil
     end
 
+    if conf['custom_metadata'].nil? || conf['custom_metadata'].strip.length == 0
```

Comment:
Should we do some validation on the configuration? Testing it is actually a KVP string?

Reply:
I added simple validation :+1: